React InstantSearch Hooks Native/getting-started/.expo-shared/assets.json

```diff
@@ -0,0 +1,4 @@
+{
+  "12bb71342c6255bbf50437ec8f4441c083f47cdb74bd89160c15e4f43e52a1cb": true,
```

Comment:
this needs to be in gitignore

Reply:
It looks like it actually can be versioned (https://stackoverflow.com/a/69546987).